### PR TITLE
import rl-algs

### PR DESCRIPTION
**Pull request by pzhokhov**
should address https://github.com/openai/baselines/issues/364, https://github.com/openai/baselines/issues/357, https://github.com/openai/baselines/issues/345,

### CODE REVIEW

File: baselines/acer/acer_simple.py [hunk 1]
```diff
@@ -13,6 +14,8 @@
 from baselines.a2c.utils import get_by_index, check_shape, avg_norm, gradient_add, q_explained_variance
 from baselines.acer.buffer import Buffer
 
+import os.path as osp
```
Comment (christopherhesse) [inline baselines/acer/acer_simple.py]: that's funny, looks like this fix https://github.com/openai/baselines/commit/569bd426297859c57c0c2ba6482f1349ac12fc01 missed the import?

Comment (pzhokhov) [reply baselines/acer/acer_simple.py]: I think so... did not seem to work without it

Comment (araffin) [reply baselines/acer/acer_simple.py]: Closes https://github.com/openai/baselines/pull/312

Comment (pzhokhov) [reply baselines/acer/acer_simple.py]: sounds good, but we'll have to export this to baselines.

**Pull request merged**